api/seedreconfig/seedreconfig.go

```diff
@@ -173,7 +173,16 @@ type ClientAuthCrypto struct {
 }
 
 type IngresssCrypto struct {
+	// THIS FIELD WAS POORLY NAMED. It should be "IngressCAPrivateKey" instead.
 	IngressCA PEM `json:"ingress_ca,omitempty"`
```

Comment:
```suggestion
	IngressCAPrivateKey PEM `json:"ingress_ca,omitempty"`
```

Actually we can just do this and keep the `json` tag the same